File: app/src/main/java/com/niceplaces/niceplaces/models/Region.java
package com.niceplaces.niceplaces.models;

public class Region {

    private String mID;
    private String mName, mCount;

    public Region(String id, String name){
        mID = id;
        mName = name;
    }

    public Region(String id, String name, String count){
        mID = id;
        mName = name;
        mCount = count;
    }

    public String getID(){
        return mID;
    }

    public String getName(){
        return mName;
    }

    public String getCount(){
        return mCount;
    }

}

File: app/src/main/java/com/niceplaces/niceplaces/models/Region.kt
package com.niceplaces.niceplaces.models

class Region {
    var iD: String
        private set
    var name: String
        private set
    var count: String? = null
        private set

    constructor(id: String, name: String) {
        iD = id
        this.name = name
    }

    constructor(id: String, name: String, count: String?) {
        iD = id
        this.name = name
        this.count = count
    }

}